nnpdf_data/nnpdf_data/filter_utils/legacy_jets_utils.py

```diff
@@ -0,0 +1,1146 @@
+"""
+Utils specifically needed to ease the implementation of the legacy jet datasets,
+these are: CMS_2JET_7TEV, CMS_1JET_8TEV, ATLAS_1JET_8TEV_R06, ATLAS_2JET_7TEV_R06
+"""
+
+import numpy as np
+import pandas as pd
+from scipy.linalg import block_diag
+import yaml
+
+def range_str_to_floats(str_range):
+    """
+    converts a string range to a list,
+    e.g. "0.5 - 1.0" --> [0.5,1.0]
+    and returns a dictionary
+    """
+    # Split the range string into two parts
+    str_nums = str_range.split('-')
+    # Convert the parts to floats
+    min = float(str_nums[0])
+    max = float(str_nums[1])
+    mid = float(f"{0.5 * (min + max):.3f}")
+    # Return a dict
+    return {"min": min, "mid": mid, "max": max}
+
+
+# ==================================================================== CMS_2JET_7TEV ====================================================================#
```

Comment:
If these are only used for a single dataset, why are they not inside the folder of the corresponding dataset?

Reply:
Yes, this is how it was before, where I had a `filter_utils.py` module in each of the datasets.

I guess that the idea is that in the dataset folders we only have a `filter.py` that produces the dataset, all of the utils instead go in `nnpdf_data/filter_utils`.
Otherwise we would have some utils in `nnpdf_data/filter_utils` and other within the dataset folder, which would be more confusing.